File: allLongestStrings/allLongestStrings.py
def allLongestStrings(a):
    ret = []
    longest = 0
    for i in range(len(a)):
        if(longest < len(a[i])):
            longest = len(a[i])
    for i in range(len(a)):
        if(longest == len(a[i])):
            ret.append(a[i])
    return ret


print(allLongestStrings(["aba", "aa", "ad", "vcd", "aba"]))

File: Completed/js/allLongestStrings/allLongestStrings.js
function allLongestStrings(a) {
    let max = 0;
    a.forEach(e => {
        max = (e.length > max) ? e.length : max;
    });
    return a.filter(e => e.length == max);
}
console.log(allLongestStrings(["aba", "aa", "ad", "vcd", "aba"]));